lib/altsvc.c

```diff
@@ -275,7 +275,7 @@ static CURLcode altsvc_out(struct altsvc *as, FILE *fp)
           "%s %s%s%s %u "
           "\"%d%02d%02d "
           "%02d:%02d:%02d\" "
-          "%u %u\n",
+          "%d %u\n",
```

Comment:
Since this value should be unsigned I think %u better illustrates the intent. It can only be 1 or 0.